templates/cluster/openstack-standalone-cp/values.schema.json

```diff
@@ -0,0 +1,405 @@
+                "type": "object",
+                "properties": {
+                "name": {
+                  "type": "string",
+                  "description": "Name of the security group to filter by"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "Optional: description for filtering"
+                },
+                "projectID": {
+                  "type": "string",
+                  "description": "Optional: project ID for filtering"
+                }
+                }
+              }
+              }
+            }
+            }
+        }
+      },
+      "worker": {
+        "type": "object",
+        "description": "Configuration of the worker instances",
+        "required": [
+          "flavor"
+        ],
+        "properties": {
+          "sshPublicKey": {
+            "type": "string",
+            "description": "SSH public key for accessing nodes"
+          },
+          "providerID": {
+            "type": ["string", "null"],
+            "description": "Unique ID for the instance provider"
+          },
+          "flavor": {
+            "type": "string",
+            "description": "OpenStack flavor for instance size"
+          },
+          "image": {
+            "type": "object",
+            "properties": {
+              "filter": {
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "type": "string",
+                    "description": "Name of the image"
+                  },
+                  "tags": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "portOpts": {
+            "type": "array",
+            "description": "Ports to be attached to the server instance",
+            "items": {
+              "type": "string"
+            }
+          },
+          "securityGroups": {
+            "type": "array",
+            "description": "Security groups to be assigned to the instance",
+            "items": {
+              "type": "object",
+              "properties": {
+              "filter": {
+                "type": "object",
+                "properties": {
+                "name": {
+                  "type": "string",
+                  "description": "Name of the security group to filter by"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "Optional: description for filtering"
+                },
+                "projectID": {
+                  "type": "string",
+                  "description": "Optional: project ID for filtering"
+                }
+                }
+              }
+              }
+            }
+            }
+        }
+      },
+      "k0smotron": {
```

Comment:
Looks like the `k0smotron` section can be removed from the schema.

Reply:
Done.